### ISSUE TITLE
Add an "Exit" item to the main menu

### ISSUE THREAD

**Issue by lenormf** [dimkanovikov/KITScenarist#892]
Hi,

It'd be convenient to have an "Exit" item in the main menu, to close the program.

Thanks!

**Comment by dimkanovikov** [comment]
Hello @lenormf, why do you think so? Why this will be better than close application by clicking on the close button on the app's top bar by your opinion?

**Comment by lenormf** [comment]
My window manager doesn't have control bars on windows, so I have to kill the process.

With a menu entry, you could set a keyboard shortcut (e.g. ^Q).

**Comment by dimkanovikov** [comment]
Hm... Does a standard shortcut like Alt+F4 or Command+Q not work for you too?

**Comment by lenormf** [comment]
No, in fact I lost some work due to ^S (save the project) not working. I haven't investigated why, I just click on the menu items to make sure - but I'd like to make sure the program exits cleanly, in case.

I'm running Scenarist on Arch, with the i3 window manager.

**Comment by dimkanovikov** [comment]
Then it looks like a general issue with shortcuts on Arch with i3. Do you use a package from the AUR?

**Comment by lenormf** [comment]
I'm using [this Dockerfile](https://github.com/dimkanovikov/KITScenarist/issues/893#issuecomment-633860065).
